lib/github/ldap/user_search/active_directory.rb

```diff
@@ -0,0 +1,49 @@
+module GitHub
+  class Ldap
+    module UserSearch
+      class ActiveDirectory < Default
+
+        # Public - Overridden from base class to set the base to "", and use the
+        # Global Catalog to perform the user search.
+        def search(search_options)
```

Comment:
This method shouldn't be public, `perform` is the public interface to this strategy, right? `search` is just called internally by `perform`.